tests/monotouch-test/AppKit/DerivedEventTest.cs

```diff
@@ -42,8 +42,13 @@ void TestDelegates (NSComboBox b)
 		[Test]
 		public void DerivedEvents_OverwriteThrows ()
 		{
-			TestOverrideThrow (false, true);
-			TestOverrideThrow (true, true);
+#if RELEASE
+			var checkTrimmedAway = TestRuntime.IsLinkAll;
```

Comment:
Nit - when `checkTrimmedAway` is used, it's always inverted - maybe it needs a different name to avoid that like maybe `throwExpected` (which is in line with the argument to `TestOverrideThrow`. So:
```
#if RELEASE
    var shouldThrow = !TestRuntime.IsLinkAll;
#else
   var shouldThrow = true;
#endif
```